Projects/Core/DesignSystem/Sources/Dialog/SMSDialog.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+import ViewUtil
+
+public extension View {
+    func smsDialog(
+        title: String,
+        description: String,
+        checkText: String,
+        cancelText: String,
+        checkAction: @escaping () -> Void = {},
+        cancelAction: @escaping () -> Void = {},
+        isShowing: Binding<Bool>
+    ) -> some View {
+        modifier(
+            SMSDialogModifier(
+                title: title,
+                description: description,
+                checkText: checkText,
+                cancelText: cancelText,
+                checkAction: checkAction,
+                cancelAction: cancelAction,
+                isShowing: isShowing
+            )
+        )
+    }
+}
+
+struct SMSDialogModifier: ViewModifier {
+    var title: String
+    var description: String
+    var checkText: String
+    var cancelText: String
+    var checkAction: () -> Void
+    var cancelAction: () -> Void
+    @Binding var isShowing: Bool
+
+    public init(
+        title: String,
+        description: String,
+        checkText: String,
+        cancelText: String,
+        checkAction: @escaping () -> Void = {},
+        cancelAction: @escaping () -> Void = {},
+        isShowing: Binding<Bool>
+    ) {
+        self.title = title
+        self.description = description
+        self.checkText = checkText
+        self.cancelText = cancelText
+        self.checkAction = checkAction
+        self.cancelAction = cancelAction
+        _isShowing = isShowing
+    }
+
+    func body(content: Content) -> some View {
+        ZStack {
+            content
+                .background(Color.blue)
+            if isShowing {
+                Rectangle().foregroundColor(Color.black.opacity(0.6))
```

Comment:
```suggestion
                Color.sms(.system(.block))
                    .opacity(0.25)
```
으로 두는거 어떤가요?

opacity는 bottomSheet가 0.25라 하긴했습니다